docs/reference/migration/migrate_8_1.asciidoc

```diff
@@ -32,9 +31,21 @@ enable <<deprecation-logging, deprecation logging>>.
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+[discrete]
+[[breaking_81_rest_api_changes]]
+==== REST API changes
 
+[[fields_api_geoshape_normalize]]
+.Fields API for geo_shape field returns the normalized geometry.
+[%collapsible]
+====
+*Details* +
+In 8.1, when retrieving a geo_shape field using the fields API, the geometry will be
+normalized if it crosses the dateline. For example if a polygon crosses the dateline,
+it will be returned as two polygons. The original polygon can still be retrieved
+from `_source`.
+====
```

Comment:
Some applications may expect unnormalized geometries. This adds an impact statement for those users.
```suggestion

*Impact* +
If your application requires unnormalized geometry objects, retrieve them from
`_source` rather than using the `fields` parameter.
====
```